app/javascript/controllers/user_controller.js

```diff
@@ -58,8 +58,13 @@ export default class UserController extends Controller {
     const rootController = this.application.controllers.find(c => c.identifier === "root")
 
     if (event.keyCode === 85) {
-      this.openUserModal()
-      rootController.setUserModalOpen(true)
+      if (rootController.isUserModalOpen()) {
+        this.closeUserModal();
+        rootController.setUserModalOpen(false);
+      } else {
+        this.openUserModal();
+        rootController.setUserModalOpen(true);
+      }
```

Comment:
Hey Etienne! Do you mind throwing this `if/else` into a `toggleUserModal` method? I expect we're going to have quite a few keyboard shortcuts and want to keep as much logic out of here as possible.

Reply:
Hi, sorry i had to do a new PR because i struggled pushing to this same branch (🤷‍♂️??), git wasn't tracking my changes. Closing this one and tagging you in the new one